command/sub8_launch/config/rise_6dof.yaml

```diff
@@ -1,8 +1,8 @@
-accel_feedforward: 0, 0, 0, 0, 0, 0
+accel_feedforward: 0, 0, 0, 0, 0, 1
```

Comment:
We should get a reference sheet going for the gains. Not sure how important it will be if we do move to sub9 but on the very high chance we don't make the move we will need it.